backend/Dockerfile

```diff
@@ -0,0 +1,31 @@
+# syntax=docker/dockerfile:1
+
+# docker build -t backend .
+
+# Use the official Node runtime as parent image
+# https://github.com/nodejs/docker-node?tab=readme-ov-file#image-variants
+FROM node:18.19.0-alpine
```

Comment:
**suggestion:** To keep versioning consistent with CI, let's use `20.11.1-alpine`